tests/end-to-end/test_able_to_sent_to_commands.py

```diff
@@ -0,0 +1,187 @@
+import time
+import json
+import random
+
+import pytest
+
+import paho.mqtt.client as mqtt
+from pymodbus.client import ModbusTcpClient
+from pymodbus.payload import BinaryPayloadDecoder
+from pymodbus.constants import Endian
+
+
+@pytest.mark.end_to_end
+def test_able_to_update_coil():
+    random_value = random.randint(1, 2) % 2 == 0
```

Comment:
I avoid random on tests
I prefer test that are deterministic

Reply:
A nicer option here might be to "parametrize" (not sure that's a real word, pytest) the coil value, so the test is run once with each possible value:

```
@pytest.mark.parametrize("coil_value",[False,True])
def test_able_to_update_coil(coil_value):
  ...
  message_dictionary = {"action": "testCoil", "value": coil_value}
```

and you get output like
```
tests/end-to-end/test_able_to_sent_to_commands.py::test_able_to_update_coil[False] PASSED
tests/end-to-end/test_able_to_sent_to_commands.py::test_able_to_update_coil[True] PASSED 
```

Reply:
that's a great suggestion